README.md

```diff
@@ -0,0 +1,94 @@
+# cancel-stuck-transactions
+
+Cancel stuck transactions on the Filecoin network.
+
+## Usage
+
+```js
+import { CancelStuckTransactions } from 'cancel-stuck-transactions'
+import timers from 'node:timers/promises'
+import fs from 'node:fs/promises'
+
+const cancelStuckTransactions = new CancelStuckTransactions({
+  // Pass a storage adapter, so that pending cancellations are persisted across
+  // process restarts
+  async store ({ hash, timestamp, from, gasPremium, nonce }) {
+    await fs.writeFile(
+      `transactions/${hash}`,
+      JSON.stringify({ hash, timestamp, from, gasPremium, nonce })
+    )
+  },
+  async list () {
+    const cids = await fs.readdir('transactions')
+    return Promise.all(cids.map(async cid => {
+      return JSON.parse(await fs.readFile(`transactions/${cid}`))
+    }))
+  },
+  async resolve (hash) {
+    await fs.unlink(`transactions/${hash}`)
+  },
```

Comment:
I find this a bit difficult to comprehend. Either these three methods should be grouped into a "store" object/interface, or else their names should indicate they work with transactions.

Reply:
They aren't working with transactions: It's not a full tx object from ethers, and it has more keys at the same time.

I've been thinking about grouping these functions in a store object, will do that then 👍